.travis/run_tests.sh

```diff
@@ -9,8 +9,6 @@ bundle install
 # allow dependent services to finish start up (e.g. ElasticSearch, Mongo)
 sleep 10
 
-bin/rake search:initialize
```

Comment:
Reviewers: Now that we removed the put_search_index_mapping calls from app.rb ([see comment](https://github.com/edx/cs_comments_service/pull/229/files#r98998466)), we can now rely on the creation of the index directly in respec, which is cleaner.